### ISSUE TITLE
Regression: sizeComponent.getValue() -> TypeError: Cannot read property 'sizeMode' of undefined

### ISSUE THREAD

**Issue by gadicc** [Famous/engine#329]
In v0.6.2, `engine/Size.js`:

```js
Size.prototype.getValue = function getValue() {
    return {
        sizeMode: this._node.value.sizeMode,  // <-- _node.value is undefined
        absolute: { ... }
        ...
}
```

**Comment by gadicc** [comment]
Workaround of course for those that need it is to retrieve sizeMode etc manually:

```js
SizeSystem.get(node.getLocation()).getSizeMode();
```

**Comment by michaelobriena** [comment]
@gadicc Woops.  Will fix.

**Comment by michaelobriena** [comment]
https://github.com/Famous/engine/pull/339

**Comment by gadicc** [comment]
Landed in c458071 as per https://github.com/Famous/engine/pull/339#issuecomment-119351670
